backend/app/api/endpoints/knowledge.py

```
@@ -1327,3 +1329,148 @@ def _update_kb_summary_after_deletion(kb_id: int, user_id: int, user_name: str):
+        has_non_text_content=chunks_data.get("has_non_text_content", False),
+        skipped_elements=chunks_data.get("skipped_elements", []),
+    )
+
+
+@document_router.delete(
+    "/{document_id}/chunks/{chunk_index}", response_model=ChunkDeleteResponse
+)
+def delete_document_chunk(
+    document_id: int,
+    chunk_index: int,
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.14.13)

Unused function argument: `current_user`

(ARG001)